scss/ionic.app.scss

```diff
@@ -40,5 +40,7 @@ $ionicons-font-path: "../lib/ionic/release/fonts" !default;
 @import "scss/pages/accounts";
 @import "scss/pages/help";
 @import "scss/pages/realitycheck";
+@import "scss/pages/profittable";
+@import "scss/pages/realaccountopening";
```

Comment:
Does it make sense to simplify them and move all sign-in & sign-up CSS to one module?